nvtabular/__init__.py

```diff
@@ -24,13 +24,17 @@
 warnings.filterwarnings("ignore", module="numba.cuda.envvars")
 
 
-ColumnGroup = column_group.ColumnGroup
+WorkflowNode = workflow.WorkflowNode
 ColumnSelector = column_selector.ColumnSelector
 Workflow = workflow.Workflow
 Dataset = io.dataset.Dataset
 
+# Provides an alias of ColumnSelector so that old usages of ColumnGroup to
+# select columns at the beginning of an operator chain don't break
+ColumnGroup = ColumnSelector
```

Comment:
mentioned this elsewhere - but how about 
```suggestion
def ColumnGroup(columns):
   warnings.warn("ColumnGroup is deprecated, use ColumnSelector instead", DeprecationWarning)
   return ColumnSelector(columns)
```